packages/react-scripts/scripts/start.js

```diff
@@ -42,6 +42,11 @@ if (!checkRequiredFiles([paths.appHtml, paths.appIndexJs])) {
   process.exit(1);
 }
 
+process.stdin.on('end', function() {
```

Comment:
Let's add a comment here (and in the other file) explaining what this does and linking to the issue.